System.IO.Ports/System.IO.Ports.nfproj

```diff
@@ -83,8 +83,15 @@
   <Import Project="..\packages\Nerdbank.GitVersioning.3.4.194\build\Nerdbank.GitVersioning.targets" Condition="Exists('..\packages\Nerdbank.GitVersioning.3.4.194\build\Nerdbank.GitVersioning.targets')" />
   <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
     <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Enable NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105.The missing file is {0}.</ErrorText>
+      <ErrorText>This project references NuGet package Nerdbank.GitVersioning.3.4.194 that is missing on this computer. Enable NuGet Package Restore to download it.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105.The missing file is {0}.</ErrorText>
```

Comment:
This is error message it's the "standard" one from .NET targets. Of course we can change it.
But, your hardcoding the version number in the message. NuGet update won't change it so we'll end up with a misleading message mentioning the same assembly name twice and with different versions.

Anyway, this PR is about Stylecop, so better keep it about that.

Reply:
I agree that this should be removed from this PR, but it is worth a seperate dissucssion about the error text (since the version number is already a manual change). I will revert it in this instance.